docs/application/tizen-studio/tizen-core/tizen-core-cli.md

```diff
@@ -115,31 +86,31 @@ tz set [options]
-|  `-T`, `--test-file-patterns=TEST-FILE-PATTERNS,...`               | Source file patterns included for test mode, format: pattern1,pattern2. |
-|  `-C`, `--chrome-path=STRING`                                      | Path of the chrome executable. |
-|  `-O`, `--chrome-simulator-options=CHROME-SIMULATOR-OPTIONS,...`   | List of options for chrome simulator, format: arg1,arg2. |
-|  `-M`, `--chrome-simulator-data-path=STRING`                       | Path of chrome simulator data. |
-|  `-L`, `--chrome-inspector-options=CHROME-INSPECTOR-OPTIONS,...`   | List of options for chrome inspector, format: arg1,arg2. |
-|  `-e`, `--chrome-inspector-data-path=STRING`                       | Path of chrome inspector data. |
+|  `-S`, `--src-file-patterns=SRC-FILE-PATTERNS,...`                 | Source file patterns excluded from build, format: pattern1, pattern2. |
+|  `-T`, `--test-file-patterns=TEST-FILE-PATTERNS,...`               | Source file patterns included for test mode, format: pattern1, pattern2. |
+|  `-C`, `--chrome-path=STRING`                                      | Path of the Chrome executable. |
+|  `-O`, `--chrome-simulator-options=CHROME-SIMULATOR-OPTIONS,...`   | List of options for Crome simulator, format: arg1, arg2. |
+|  `-M`, `--chrome-simulator-data-path=STRING`                       | Path of Chrome simulator data. |
+|  `-L`, `--chrome-inspector-options=CHROME-INSPECTOR-OPTIONS,...`   | List of options for Chrome inspector, format: arg1, arg2. |
+|  `-e`, `--chrome-inspector-data-path=STRING`                       | Path of Chrome inspector data. |
 |  `-o`, `--optimize={true|false}`                                   | Size optimization of `wgt` for web workspace. |
-|  `-P`, `--projects=STRING`                                         | List of projects in the workspace and their dependencies, format: proj1=[dep1:dep2:dep3],proj2. |
+|  `-P`, `--projects=STRING`                                         | List of projects in the workspace and their dependencies, format: proj1=[dep1:dep2:dep3], proj2. |
 
 **Examples:**
 
-- To set llvm, arm and Release type for build, use following command:
+- To set llvm, arm and release type for build, use the following command.
   
   Windows&reg;:
   ```
-  > tz set -A arm -c llvm -b release -W C:\Users\native_workspace
+  > tz set -A arm -c llvm -b release -w C:\Users\workspace
   ```
 
-  Ubuntu and Mac:
+  Ubuntu and macOS:
   ```
-  $ tz set -A arm -c llvm -b release -W ~/native_workspace
+  $ tz set -A arm -c llvm -b release -w ~/workspace
   ```
 
-- Displays the list of all configurations for the given workspace. Individual options can be displayed by passing the required option flag.
+- Displays the list of all configurations for the given workspace. Individual options can be displayed by passing the required option flag:
 
 ```
 tz get [options]

```

Comment:
@vivek-ellur,
Line 115: Please add two more spaces before the code tags (```). The correction suggestion on the line is not working properly. Hence, the comment. 
The change has been suggested to align the code with the bullet point. Please look into it. 
```suggestion
  tz get [options]

```